assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -104,6 +104,7 @@ data "template_file" "controller-configs" {
     ssh_keys              = jsonencode(var.ssh_keys)
     k8s_dns_service_ip    = cidrhost(var.service_cidr, 10)
     cluster_domain_suffix = var.cluster_domain_suffix
+    controllers           = var.controller_count
```

Comment:
The node_private_cidr can be added here to use in the iptables rule, right? I mean, if we need only node traffic to the 7443 port

Reply:
Yes, we could do that, but as said, we only listen on private IP address, so that would be no-op anyway for external traffic. I'm not sure if I want to make it more complex at this point.

Reply:
Oh, okay!